### PR TITLE
Metaknight changes

### PR DESCRIPTION
**Pull request by techyCoder81**
mostly nerfs to metaknight. A few sidegrade changes. Goes with romfs changes that reanimate fair, though not *necessary* for testing per se.

### CODE REVIEW

**Comment by Riah321** [comment]
Changelog please!

**Comment by MadVillainLordQ** [comment]
LGTM

Notes:
- I still think this fair is still very strong but its hard to know how effective DI will be against it.

**Comment by SkewedAskew** [comment]
Lgtm

- Was not able to test SDI but it definitely seems like fair trains are gone
- Drill Rush has a strange quirk where connecting with the move and landing after performing any followup action will still give him the landing lag of Drill Rush as opposed to whatever he landed with

File: fighters/metaknight/src/meta_quick.rs [hunk 1]
```diff
@@ -220,9 +220,9 @@ unsafe fn check_apply_speeds(fighter: &mut smash::lua2cpp::L2CFighterCommon) {
     // handle speed application once
     if VarModule::is_flag(fighter.object(), vars::metaknight::instance::META_QUICK_NEED_SET_SPEEDS) {
         if VarModule::get_int(fighter.object(), vars::common::instance::GIMMICK_TIMER) > 0 {
-            apply_status_speed_mul(fighter, 1.25);
+            apply_status_speed_mul(fighter, 1.30);
```
Comment (jobrien97) [inline fighters/metaknight/src/meta_quick.rs:223]: Fine for now, but should definitely be stored in ParamModule

File: fighters/metaknight/src/acmd/aerials.rs [hunk 2]
```diff
@@ -61,32 +61,36 @@ unsafe fn metaknight_attack_air_f_game(fighter: &mut L2CAgentBase) {
...
-        ATTACK(fighter, 1, 0, Hash40::new("top"), 1.5, 367, 100, 40, 0, 6.5, 0.0, 0.0, 9.0, Some(0.0), Some(11.0), Some(9.0), 0.5, 1.0, *ATTACK_SETOFF_KIND_THRU, *ATTACK_LR_CHECK_F, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_A, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
+        FT_MOTION_RATE(fighter, 3.000);
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 1.5, 365, 100, 40, 0, 6.5, 0.0, 6.0, 15.5, None, None, None, 0.5, 1.0, *ATTACK_SETOFF_KIND_THRU, *ATTACK_LR_CHECK_F, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_A, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
```
Comment (jobrien97) [inline fighters/metaknight/src/acmd/aerials.rs]: First hit hitlag mul should be changed to 1.0 now that it is a 2-hit

**Review by jobrien97 — APPROVED** [review]
lgtm

File: fighters/metaknight/src/meta_quick.rs (hunk 1 above)
Comment (techyCoder81) [reply fighters/metaknight/src/meta_quick.rs:223]: Agree

**Pull request merged**